### PR TITLE
Add note about the increased minimum requirements

### PR DESCRIPTION
**Pull request by TimWolla**

### CODE REVIEW

File: pages/migration/wsc-53/migration_wsc-53_php.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+folder: migration/wsc-53
+---
+
+## Minimum requirements
+
+The minimum requirements have been increased to the following:
+
+- **PHP:** 7.2
+- **MySQL:** 5.7 / 8.0
+- **MariaDB:** 10.1
```
Comment (Gravatronics) [inline pages/migration/wsc-53/migration_wsc-53_php.md:14]: Shouldn't there also be the patch version as 7.2.0 is a PHP 7.2, but we require 7.2.24?

**Review by Gravatronics — CHANGES_REQUESTED** [review]

File: pages/migration/wsc-53/migration_wsc-53_php.md (hunk 1 above)
Comment (TimWolla) [reply pages/migration/wsc-53/migration_wsc-53_php.md:14]: This is the developer documentation, not the end user documentation. No functional changes are expected between 7.2.0 and 7.2.24. I'm intentionally only putting the branch here for clarity.

Comment (Gravatronics) [reply pages/migration/wsc-53/migration_wsc-53_php.md:14]: I still think that we should specify system requirements as precisely as possible, in particular as adding just another digit does not really change readability.

Comment (dtdesign) [reply pages/migration/wsc-53/migration_wsc-53_php.md:14]: Having another place where we document these requirements isn't a bad idea, makes it easier to find the exact requirements.

**Review by Gravatronics — APPROVED** [review]

**Pull request merged**